libraries/AP_Math/AP_Math.cpp

```diff
@@ -473,3 +473,9 @@ float fixedwing_turn_rate(float bank_angle_deg, float airspeed)
     bank_angle_deg = constrain_float(bank_angle_deg, -80, 80);
     return degrees(GRAVITY_MSS*tanf(radians(bank_angle_deg))/MAX(airspeed,1));
 }
+
+// convert degrees farenheight to Kelvin
+float degF_to_Kelvin(float temp_f)
+{
+    return (temp_f + 459.67) * 0.55556;
```

Comment:
Surely in-lining this is fewer bytes than making a call?

Reply:
not tested, but i'd guess not inlining is better